clippy_lints/src/lib.rs

```diff
@@ -746,6 +748,7 @@ pub fn register_plugins(reg: &mut rustc_plugin::Registry<'_>, conf: &Conf) {
         matches::MATCH_REF_PATS,
         matches::MATCH_WILD_ERR_ARM,
         matches::SINGLE_MATCH,
+        mem_replace::MEM_REPLACE_OPTION_WITH_NONE,
```

Comment:
This also has to be added to the `clippy::all` Lint group. You can run the `utils/update_lints.py` script to do this automatically.

Reply:
Added

Reply:
Yeah that's ok. Probably another PR didn't update the README and the CHANGELOG files, which is also ok. `lib.rs` is the important file here.